targets/rpi4.docker

```diff
@@ -48,12 +48,25 @@ RUN __renv__ __rdistfiles__ __rccache__ \
         --no-merged-usr \
         --cache-dir=/var/cache/distfiles \
         --components="main,contrib,non-free" \
-        --include="__debootstrap_dinclude__ __debootstrap_include__"\
-ifdef(`__debootstrap_exclude__', `dnl
-        --exclude="__debootstrap_exclude__" \
-')dnl
+        --include="__debootstrap_dinclude__"\
         "__debootstrap_release__" ${SYSROOT} "__debootstrap_url__" || ( cat ${SYSROOT}/debootstrap/debootstrap.log; exit 1 )
-    
+
+ifelse(index(__debootstrap_include__, `1'),0,`dnl
+COPY ./repos/debian/rpi4/config/multistrap.conf ${SYSROOT}/
+
+RUN __renv__ __rdistfiles__ __rccache__ \
+    cp -f /etc/resolv.conf ${SYSROOT}/etc/resolv.conf && \
+    sed -i "/Install-Recommends=false/ s|$|;\n\$config_str .= \" -o Acquire::AllowInsecureRepositories=yes\"|" \
+     ${SYSROOT}/usr/sbin/multistrap && \
+    chroot ${SYSROOT} multistrap -a ${TPARCH} -d /multistrap -f /multistrap.conf
+
+RUN __renv__ __rdistfiles__ __rccache__ \
+    mkdir /home/debian && \
```

Comment:
I would avoid touching anything in `/` much, can we move this to `/tmp` or `/var` and back? If we need a small Debian installation to run `multistrap` from, perhaps we could simply `deboostrap` in `/var/cache` and not move anything afterwards.